### PR TITLE
samples: basic: blinky: eGPIO PoC

### PR DESCRIPTION
**Pull request by jaz1-nordic**

### CODE REVIEW

File: samples/basic/blinky/sample.yaml [hunk 1]
```diff
@@ -10,3 +10,31 @@ tests:
...
+    platform_allow: nrf54l15pdk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15pdk/nrf54l15/cpuapp
+    tags:
+      - LED
+      - gpio
+      - ipc
+    filter: dt_enabled_alias_with_parent_compat("led4", "gpio-leds")
+    depends_on: gpio
+    harness:
```
Comment (nordic-piks) [inline samples/basic/blinky/sample.yaml:24]: Please change to `harness: console`
Current configuration fails and twister cannot start any testing.

**Comment by nordic-piks** [comment]
nrf54l15pdk/nrf54l15/cpuapp samples/drivers/mbox/sample.drivers.mbox.nrf54l15_no_multithreading PASSED
nrf54l15pdk/nrf54l15/cpuapp samples/subsys/ipc/ipc_service/icmsg/sample.ipc.icmsg.nrf54l15 PASSED
6/6 nrf54l15pdk/nrf54l15/cpuapp samples/subsys/ipc/ipc_service/icmsg/sample.ipc.icmsg.nrf54l15_no_multithreading  FAILED Timeout (device: 1057743738, 68.087s)
14:10:01  INFO    - /jenkins_cache/test-zephyr-sanitycheck/zephyr/zephyr/twister-out/nrf54l15pdk_nrf54l15_cpuapp/samples/subsys/ipc/ipc_service/icmsg/sample.ipc.icmsg.nrf54l15_no_multithreading/handler.log
```
14:10:01  ERROR   - *** Booting Zephyr OS build v3.7.0-rc1-437-g87278dcc74c8 ***
14:10:01  I: IPC-service HOST demo started
14:10:01  E: ***** USAGE FAULT *****
14:10:01  E:   Stack overflow (context area not valid)
14:10:01  E: r0/a1:  0x0000710e  r1/a2:  0x20018008  r2/a3:  0x00000000
14:10:01  E: r3/a4:  0x00007110 r12/ip:  0x0000a5df r14/lr:  0x0000103d
14:10:01  E:  xpsr:  0x09000000
14:10:01  E: Faulting instruction address (r15/pc): 0x00001044
14:10:01  E: >>> ZEPHYR FATAL ERROR 2: Stack overflow on CPU 0
14:10:01  E: Current thread: 0 (unknown)
14:10:01  E: Halting system
```
2nd and3rd execution:
nrf54l15pdk/nrf54l15/cpuapp samples/subsys/ipc/ipc_service/icmsg/sample.ipc.icmsg.nrf54l15_no_multithreading  FAILED Timeout (device: 1057717622, 67.787s)
14:11:38  INFO    - /jenkins_cache/test-zephyr-sanitycheck/zephyr/zephyr/twister-out/nrf54l15pdk_nrf54l15_cpuapp/samples/subsys/ipc/ipc_service/icmsg/sample.ipc.icmsg.nrf54l15_no_multithreading/handler.log
```
14:11:38  ERROR   - *** Booting Zephyr OS build v3.7.0-rc1-437-g87278dcc74c8 ***
14:11:38  I: IPC-service HOST demo started
14:11:38  I: Ep bounded
14:11:38  I: Perform sends for 1000 [ms]
14:11:38  I: Sent 2003473 [Bytes] over 1000 [ms]
14:11:38  I: Wait 500ms. Let remote core finish its sends
14:11:38  I: Received 2003473 [Bytes] in total
14:11:38  I: IPC-service HOST demo ended
```

**Comment by nordic-piks** [comment]
In general test can be run using this job - please notice parameters:
https://jenkins-ncs.nordicsemi.no/job/latest/job/test-fw-twister-upstream-zephyr/job/customizable/job/main/338/parameters/

**Comment by jaz1-nordic** [comment]
Changes applied here: https://github.com/nrfconnect/sdk-nrf/pull/16393

**Pull request closed without merging**